audioswitch/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <uses-permission android:name="android.permission.BLUETOOTH"  android:maxSdkVersion="30"/>
-    <uses-permission android:name="android.permission.BLUETOOTH_CONNECT" />
     <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
```

Comment:
are you sure this should be removed, the bluetooth permissions are asked/granted to via user interaction..  this just adds the option to the system menu associated with the app.

Reply:
Removing these permission will allow developers to decide whether to add Bluetooth permission or not.  Otherwise they come with the use of the AudioSwitch. I guess the main concern was the BLUETOOTH_CONNECT which is a runtime permission, that was expressed in the original ticket filed regarding the bluetooth permissions; apps that don't want to use bluetooth capability will still need to request the permission.

Reply:
ok, lets make it more explicit in the README.